packages/preview/ecnu-math-hwk/0.1.0/README.md

```diff
@@ -0,0 +1,46 @@
+# ECNU Math Homework 
+
+为华东师范大学写的作业模板，主要针对 问题-解答/证明的形式。
+
+## 特点
+- 支持中英文
+- 格式简约美观
+- 为每个问题中的行间公式进行编号，可以直接引用。
+- 大大的校徽水印
+
+适用于提交 问题-解答/证明 类型的 pdf 作业
+
+## 预览
+| 1                    | 2                  | 3                |
+| ----------------------- | --------------------- | --------------------- |
+| ![1](pic/1.png) | ![2](pic/2.png) | ![3](pic/3.png) |
```

Comment:
It would be nice to have actual image descriptions and not just numbers. They don't have to be very detailed but should still give an idea of what the image shows.